### DISCUSSION TITLE
'Try it' not responding

### DISCUSSION THREAD

**Discussion by markoarnauto** [GitbookIO/community#672] [4. Open API]
Anyone else facing issues with the 'try it' feature of OpenAPI blocks?
By looking into the web console it seems to have cors issues. 

Trying the exact same OpenAPI specs on scalar.com works as expected.
P.s. I use a custom domain if that impact the behavior.

**Comment by aikoii** [comment]
Hey @markoarnauto! I know we could have some clearer messaging on this, but you need to make sure that the CORS headers are set correctly so they can be fetched by our servers. Let me know if you have any issues.
